cmd/firebolt/main.go

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	sdk "github.com/conduitio/conduit-connector-sdk"
+
+	firebolt "github.com/conduitio-labs/conduit-connector-firebolt"
+	"github.com/conduitio-labs/conduit-connector-firebolt/destination"
+	"github.com/conduitio-labs/conduit-connector-firebolt/source"
+)
+
+func main() {
+	sdk.Serve(sdk.Connector{
+		NewSpecification: firebolt.Specification,
+		NewSource:        source.New,
+		NewDestination:   destination.New,
+	})
```

Comment:
You may want to have the sdk.Connector{...} part in a constant, which you'll use in the acceptance tests (it's important that they are the same).

Reply:
done